src/containers/App/App.js

```diff
@@ -38,8 +39,10 @@ class App extends React.Component {
         const {singleClusterMode, clusterName} = this.props;
         return (
             <AsideNavigation>
-                <Content singleClusterMode={singleClusterMode} clusterName={clusterName} />
-                <div id="fullscreen-root"></div>
+                <ErrorBoundary>
+                    <Content singleClusterMode={singleClusterMode} clusterName={clusterName} />
+                    <div id="fullscreen-root"></div>
+                </ErrorBoundary>
```

Comment:
The goal of boundary to allow user continue working in the app, so not everything is wrapped. 

So, on the one hand this doesn't cover possible problems with `Navigation`, `ThemeProvider` and `Authentication`. On the other hand, without them it would be just a page with some strange text, not really better, than the white screen we have currently